src/Resources/skeleton/controller/ControllerWithTwig.twig.php

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html.twig' %}
```

Comment:
If `base.html.twig` template does not exists then the generated code is broken, which is bad experience :)

Reply:
I mean, after removing the default base template provided by Twig-Bundle recipe.